tests/e2e/vsan_stretched_cluster.go

```diff
@@ -260,7 +260,8 @@ var _ = ginkgo.Describe("[vsan-stretch-vanilla] vsan stretched cluster tests", f
 		6.	Bring primary site up and wait for testbed to be back to normal
 		7.	Delete all objects created in step 2 and 5
 	*/
-	ginkgo.It("Primary site down", ginkgo.Label(p0, vsanStretch, block, vanilla, wcp, tkg, primaryCentric), func() {
+	ginkgo.It("Primary site down", ginkgo.Label(p0, vsanStretch, block, vanilla, wcp, tkg, primaryCentric,
```

Comment:
All vsan-stretch Block vanilla tests works for File as well. Please add File label as well